docker/Dockerfile

```diff
@@ -1,9 +1,34 @@
+FROM eclipse-temurin:11 as jre-build
 
-FROM openjdk:11.0.8-jre-slim-buster
+# Create a custom Java runtime
+RUN JAVA_TOOL_OPTIONS="-Djdk.lang.Process.launchMechanism=vfork" $JAVA_HOME/bin/jlink \
+         --add-modules ALL-MODULE-PATH \
+         --strip-debug \
+         --no-man-pages \
+         --no-header-files \
+         --compress=2 \
+         --output /javaruntime
 
-COPY ethsigner /opt/ethsigner/
+FROM ubuntu:latest
```

Comment:
What version should we use? Using latest as the code scanning is warning about seems like a bad idea.

Reply:
The code scanning was warning about previous buster image we were using. The above approach is used by Teku and Web3Signer as well. Basically we use latest ubuntu image (which has no known security vulnerabilities) and then copy over JDK on it from JDK's docker image. (hence two "FROM" in docker file).